build/ci/do_bats.sh

```diff
@@ -212,6 +239,10 @@ do_process_branch() {
     echo "BATS_ERROR_FILES_COUNT=${fail_files}" >> "${outfile}"
     BATS_ERROR_COUNT=${fail_all}
     echo "BATS_ERROR_COUNT=${BATS_ERROR_COUNT}" >> "${outfile}"
+    echo "BATS_ERROR_FILES_TIMED_OUT=${timed_out_files}" >> "${outfile}"
+    echo "BATS_ERROR_FILES_HARDLY_TIMED_OUT=${hardly_timed_out_files}" >> "${outfile}"
```

Comment:
For the users I would:
- add also the timed out tests to the error files
- have only BATS_ERROR_FILES_TIMED_OUT with all timeouts